packages/oauth/src/providers/linkedin.ts

```diff
@@ -82,7 +74,7 @@ export const linkedin = <_Auth extends Auth>(auth: _Auth, config: Config) => {
 				{
 					clientId: config.clientId,
 					redirectUri: config.redirectUri,
-					scope: ["r_liteprofile", ...scopeConfig]
+					scope: scopeConfig
```

Comment:
Isn't `profile` scope required?

```suggestion
					scope: ["profile", ...scopeConfig]
```

Reply:
I think we should mention it in the document and leave it configurable instead. We don't know if MS change it in the future or not (like `r_liteprofile`.

Reply:
Isn't the `profile` scope required to get the user info?

Reply:
yeah, `profile` is _Required to retrieve the member's lite profile including their id, name, and profile picture. Agreed with you.